### ISSUE TITLE
[Feature Request] Packages list

### ISSUE THREAD

**Issue by Dallas62** [balderdashy/www.sailsjs.org#62]
Hi!

It would be nice if we could see all the packages (hooks, adapters, ...) for Sails like you do on http://node-machine.org/

Maybe by using npm API, and using specifics keywords to find them...

tks!

**Comment by tjwebb** [comment]
Yea something like http://react-components.com which just scrapes all npm modules with "react-component" keyword. I'd love to build this but I just don't have the time. We'd be grateful if someone in the community could help set this up.

**Comment by sailsbot** [comment]
Thanks for posting, @Dallas62.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
